src/com/MC/ticTacToe/helper/TicTacToeHelper.java

```diff
@@ -0,0 +1,69 @@
+package com.MC.ticTacToe.helper;
+
+public class TicTacToeHelper {
```

Comment:
"Helper" seems very generic. This will get bloated with all kind of helper functions over time as this doesn't have well defined responsibility defined. Follow SRP with clear responsibilities assigned to each class.